### PR TITLE
Improve communication of the stale bot.

### PR DESCRIPTION
**Pull request by karst**
This PR improves some of the messages sent by the stale bot, making it less of a bother and communicate better that something can be re-opened if something recurs of if someone wants to work on a stale PR again. 

I have also upped the message from stale PR from two weeks to a month, so it's a bit less spammy as I noticed in the past weeks the same prs are getting the message over and over, due to devs still being busy with other things (like getting ready for a release, but it can also be an outside influence). If I need to revert this please let me know.

Use this PR if you want, but close this if you feel it's unnecessary.

### CODE REVIEW

File: .github/workflows/stale-prs.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ jobs:
         with:
           days-before-issue-stale: -1
           days-before-issue-close: -1
-          days-before-pr-stale: 14
+          days-before-pr-stale: 31
           days-before-pr-close: 14
           stale-pr-label: 'stale-pr'
-          stale-pr-message: 'This pull request is stale and will be closed in 14 days if no action is taken. To keep it open, leave a comment or remove the `stale-pr` label. If you''re awaiting feedback from a developer, please send us a reminder (either here or on [Discord](https://discord.gg/ZXZd8D8)).'
-          close-pr-message: 'This pull request was closed due to inactivity.'
+          stale-pr-message: 'This pull request has been marked as stale and will be closed in 14 days if no action is taken. To keep it open, leave a comment or remove the `stale-pr` label. If you''re awaiting feedback from a developer, please send us a reminder (either here or on [Discord](https://discord.gg/ZXZd8D8)).'
```
Comment (fidwell) [inline .github/workflows/stale-prs.yml:17]: Should this read 31 days instead of 14 days?

**Review by ZehMatt — CHANGES_REQUESTED** [review]
I'm fine with the rephrasing and increased tolerance for PRs, the backtrace issue change should be reverted as people don't really engage in those backtrace issues.

File: .github/workflows/stale-prs.yml [hunk 2]
```diff
@@ -11,8 +11,8 @@ jobs:
...
+          close-pr-message: 'This pull request has been closed due to inactivity. If you do wish to continue with this PR or are still waiting for review, don''t hesitate to reach out to one of the devs by tagging them here, or on [Discord](https://discord.gg/ZXZd8D8), so we can re-open this PR.'
```
Comment (tupaschoal) [inline .github/workflows/stale-prs.yml:18]:
```suggestion
          close-pr-message: 'This pull request has been closed due to inactivity. If you do wish to continue with this PR or are still waiting for review, don't hesitate to reach out to one of the devs by tagging them here, or on [Discord](https://discord.gg/ZXZd8D8), so we can re-open this PR.'
```

Comment (Broxzier) [reply .github/workflows/stale-prs.yml:18]: The double quote is for escaping an actual quote character within the string.

Comment (Broxzier) [inline .github/workflows/stale-prs.yml:18]: A slight rewording:

```suggestion
          close-pr-message: 'This pull request has been closed due to inactivity. If you want to continue with this, or are awaiting a review, don''t hesitate to reach out to a developer. We'll gladly re-open this PR! You can tag us here, or send us a message on [Discord](https://discord.gg/ZXZd8D8).'
```

Comment (tupaschoal) [reply .github/workflows/stale-prs.yml:18]: Ah, my bad

File: .github/workflows/stale-backtrace-issues.yml [hunk 3]
```diff
@@ -16,5 +16,5 @@ jobs:
           only-issue-labels: 'backtrace.io'
           stale-issue-label: 'stale-backtrace'
           stale-issue-message: 'This issue is stale and will be closed tomorrow if no action is taken. To keep it open, leave a comment or remove the `stale-backtrace` label.'
-          close-issue-message: 'This issue was closed due to inactivity.'
+          close-issue-message: 'This issue was closed due to inactivity. If this issue does recur, please let a developer know.'
```
[the remark this answers is not in the document]
Comment (karst) [reply .github/workflows/stale-backtrace-issues.yml]: Done

**Comment by ZehMatt** [comment]
Looks like some error slipped in here, there is some bad yml format.

**Comment by Broxzier** [comment]
Fixed in 50242c21608b16783276614cf077e943c09242f4

**Comment by ZehMatt** [comment]
Thanks

**Pull request merged**